### DISCUSSION TITLE
Use of fqa ('Alternate Translation') in BSB

### DISCUSSION THREAD

**Discussion by klassenjm** [usfm-bible/tcdocs#87] [General]
We are working toward a BSB (Berean Standard Bible) text in USFM markup, as a model/exemplar text.

This discussion is intended to help us draw some conclusions about the uses of `\fqa` ([docs](https://docs.usfm.bible/usfm/3.1/char/notes/footnote/fqa.html)) in BSB footnotes. We are carefully following the [typeset layout](https://bereanbible.com/bsb-book.pdf) provided on the BSB website, attempting to adequately mark all of the textual elements which should be identified as a matter of normal practice, or which have a formatted presentation in the BSB layout (i.e. beyond normal footnote text).

Following are some examples from the book of Exodus. In **Examples 1 and 2**, the text marked with `\fqa` is quite clearly an alternate translation. In **Examples 3-5**, the typeset layout shows that the BSB team intends to identify some footnote text, but how that should be done as an model is undecided. There are 2 options shown for each example -- an option where the text is marked with `\em ... \em*` ([emphasis](https://docs.usfm.bible/usfm/3.1/char/features/em.html)), and an option with `\fqa` ([alternate translation](https://docs.usfm.bible/usfm/3.1/char/notes/footnote/fqa.html)). Using \fqa has merit in terms of simplicity. However, a question has arisen as to whether these texts (and others like them) are actually 'alternate translations' and whether they should be marked as such. For example: in Example 3, "to lift out" is not a possible alternative in the scripture text for 'Moses'. 

If we broaden the definition for `\fqa`, we can use it more widely, and simplify footnote markup. However, we would lose clarity in some footnotes about where there are truly alternate texts being contrasted.

-----
Example 1: **EXO 1:10**
```
... fight against us, and leave the country.\f + \fr 1:10 \ft Or \fqa and take the country\f*”
```
-----
Example 2: **EXO 2:3**
```
... she got him a papyrus basket\f + \fr 2:3 \ft The Hebrew can also mean \fqa ark\ft ; 
also in verse 5; see \+xt Genesis 6:14\+xt*.\f*
```
-----
Example 3: **EXO 2:10**
_Option 1_
```
... She named him Moses\f + \fr 2:10 \fq Moses \ft sounds like a Hebrew term 
that means \+em to lift out\+em*.\f* and explained, “I drew him out of the water.”
```
_Option 2_
```
... She named him Moses\f + \fr 2:10 \fq Moses \ft sounds like a Hebrew term 
that means \fqa to lift out\ft .\f* and explained, “I drew him out of the water.”
```
-----
Example 4: **EXO 2:18**
_Option 1_
```
When the daughters returned to their father Reuel,\f + \fr 2:18 \fq Reuel \ft was 
also called \+em Jethro\+em*; see \+xt Exodus 3:1\+xt*.\f* he asked them, 
“Why have you returned so early today?”
```
_Option 2_
```
When the daughters returned to their father Reuel,\f + \fr 2:18 \fq Reuel \ft was 
also called \fqa Jethro\ft ; see \+xt Exodus 3:1\+xt*.\f* he asked them, 
“Why have you returned so early today?”
```
-----
Example 5: **EXO 2:22**
_Option 1_
```
And she gave birth to a son, and Moses named him Gershom,\f + \fr 2:22 \fq Gershom 
\ft sounds like the Hebrew for \+em foreigner\+em*.\f* saying, “I have become a 
foreigner in a foreign land.”
```
_Option 2_
```
And she gave birth to a son, and Moses named him Gershom,\f + \fr 2:22 \fq Gershom 
\ft sounds like the Hebrew for \fqa foreigner\ft .\f* saying, “I have become a 
foreigner in a foreign land.”
```

**Comment by LBCBoot** [comment]
\fqa is used with "or", or "literally"

\fqa is sometimes used to escape the \fq check

Recommendation to distinguish between \+em .... \+em* and \fqa by using \fqa just for the alternative readings (with or or literally)

**Comment by KentSpiel** [comment]
After reviewing the examples and the typesetting cues, I would suggest that **Option 1 markups** are preferred for the BSB example text, based on the intended meaning and proper use of the USFM markers:

Tag | Intended Use
-- | --
`\fqa` | Alternate translation — a viable rendering of the Scripture text itself
`\+em` | Emphasis — used for explanations, etymology, or stylistic highlighting

That said, if a project values **markup simplicity and internal consistency,** it is reasonable to adopt a broader policy for using `\fqa `  provided that **the approach is clearly documented and applied consistently throughout the text.**
